tests/kserve_models_web_application_test.sh

```diff
@@ -0,0 +1,169 @@
+#!/bin/bash
+set -euxo pipefail
+
+# KServe Models Web Application Test Script
+# Tests the models-web-application API functionality with kubectl-deployed InferenceServices
+
+KF_PROFILE=${1:-kubeflow-user-example-com}
+TOKEN="$(kubectl -n $KF_PROFILE create token default-editor)"
+BASE_URL="localhost:8080/kserve-endpoints"
+
+echo "=========================================="
+echo "KServe Models Web Application Test"
+echo "Profile: ${KF_PROFILE}"
+echo "=========================================="
+
+# Pre-Test Setup: Configure models-web-application to disable authentication for testing
+kubectl patch configmap kserve-models-web-app-config -n kubeflow \
+  --type merge \
+  -p '{"data":{"APP_DISABLE_AUTH":"True"}}'
+
+kubectl rollout restart deployment kserve-models-web-app -n kubeflow
+kubectl rollout status deployment kserve-models-web-app -n kubeflow --timeout=120s
+
+# Pre-Test Setup: Create RBAC for default-editor to access InferenceServices
```

Comment:
Why do you add this? the default editor already should have the permissions to modify kserve inferenceservices